src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person | null,
```

Comment:
can you explain, in which case you expect person is null? in your usage of PersonLink component, you check person value, so it can't be null 
<img width="290" alt="Screenshot 2023-08-06 at 9 07 43 PM" src="https://github.com/mate-academy/react_people-table-basics/assets/60295799/9a393995-28b5-4dac-9fe6-64f3397edd76">